osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -45,15 +48,16 @@ public DrawableJudgement(Judgement judgement, DrawableHitObject judgedObject)
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load(OsuConfigManager config, OsuColour colours)
         {
             this.colours = colours;
+            hitErrorJudgements = config.GetBindable<bool>(OsuSetting.HitErrorJudgments);
 
             Child = new SkinnableDrawable($"Play/{Judgement.Result}", _ => JudgementText = new OsuSpriteText
             {
                 Text = Judgement.Result.GetDescription().ToUpperInvariant(),
                 Font = @"Venera",
-                Colour = judgementColour(Judgement.Result),
+                Colour = hitErrorJudgements ? JudgementColour(Judgement) : judgementColour(Judgement.Result),
```

Comment:
As far as I see, this is the only `hitErrorJudgements` usage. Hence, it may be converted to a local variable.